forte/data/data_pack.py

```diff
@@ -867,25 +868,54 @@ def get_data(
         valid_context_ids: Set[int] = self._index.query_by_type_subtype(
             context_type_
         )
+
         if context_components:
             valid_component_id: Set[int] = set()
             for component in context_components:
                 valid_component_id |= self.get_ids_by_creator(component)
             valid_context_ids &= valid_component_id
 
+        def get_annotation_list(c_type):
```

Comment:
The docstrings for these two functions are a bit weak, they are inner functions but proper documentation can help users or developers, let's explain the input and output type properly.